packages/core/src/app/shipping/NewConsignment.tsx

```diff
@@ -74,8 +74,6 @@ const NewConsignment = ({
             if (error instanceof AssignItemFailedError) {
                 onUnhandledError(error);
             }
-        } finally {
-            toggleAllocateItemsModal();
```

Comment:
This line creates a memory leak and not needed to close the modal actually.

Reply:
Just identified the memory leak is due to line 74 and not this one.

Reply:
Do we still need this line to close the modal?

Reply:
We don't need this line can get rid of it, just that it does not solve the issue.

Reply:
Adding it back as it failed the test, the modal was not closed. 🤦🏻‍♂️
We can address the memory issue separately.
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/f72c7296-3cfd-4888-a283-e4407b64b022">